packages/rspack-test-tools/tests/configCases/externals/item-value-object/rspack.config.js

```diff
@@ -3,14 +3,20 @@ const { CopyRspackPlugin } = require("@rspack/core");
 /** @type {import("@rspack/core").Configuration} */
 module.exports = {
 	entry: "./index.js",
+	output: {
+		library: {
+			type: "umd",
+		}
+	},
 	externals: {
 		lodash: {
 			root: "_",
 			commonjs: "./lodash.js",
+			commonjs2: "./lodash.js",
 			amd: "./lodash.js"
 		}
 	},
-	externalsType: "commonjs",
+	externalsType: "umd",
```

Comment:
External object value is only for umd, should not use it with other types, can just use `lodash: "./lodash.js"` as replacement